build.psm1

```diff
@@ -240,7 +245,7 @@ function Start-ScriptAnalyzerBuild
         }
 
         $buildConfiguration = $Configuration
-        if ((3, 4) -contains $PSVersion) {
+        if ((3, 4, 6, 7) -contains $PSVersion) {
```

Comment:
Because only 7 is special, wouldn't code be simpler if we didn't add 6 so that we would only need the PSVersion7 constant in the csproj file but not PSVersion6?

Reply:
possibly, but i thought it would be better to be more complete and treat *5* as the only special one. Since assemblies are put in special directories, i thought it was best to provide constants that could be used